wasm/stats.cc

```diff
@@ -0,0 +1,125 @@
+#include <string>
```

Comment:
I thought I left this comment but might have forgotten.

Does it make sense to move this file under `envoy/wasm/src/` and the precompiled binary under `envoy/wasm/bin/`?

Reply:
Right now it's easier to keep the source and binary in the same place. Once we have the build integrated with `make docker-build-osm-controller` then splitting up the source and binary would be easier. Until then, keeping them separate would just require more manual steps.

Reply:
Actually having the prebuilt docker image was the only blocker to integrating the build. I'll remove the compiled binary from this PR and have the build place it directly alongside the osm-controller binary in a future PR.